airflow/utils/docs.py

```diff
@@ -16,10 +16,12 @@
 # under the License.
 from __future__ import annotations
 
-try:
-    import importlib_metadata
-except ImportError:
-    from importlib import metadata as importlib_metadata  # type: ignore[no-redef]
+import sys
+
+if sys.version_info >= (3, 10):
+    from importlib import metadata
```

Comment:
This is - as I understand, more of a precaution coming from:

https://docs.python.org/3.12/library/importlib.metadata.html 

> Changed in version 3.10: importlib.metadata is no longer provisional.

Reply:
https://docs.python.org/3.12/library/importlib.metadata.html#distribution-metadata

> Changed in version 3.10: The Description is now included in the metadata when presented through the payload. Line continuation characters have been removed.

But maybe it is not affect us

Reply:
Any more comments ? Or should we merge it for 2.9?